doc/guides/bluetooth/mesh-pics.rst

```diff
@@ -0,0 +1,312 @@
+.. _mesh-pics:
+
+MESH PICS
+*********
+
+PTS version: 7.4.1
+
+\* - different than PTS defaults
+
+\^ - field not available on PTS
+
+M - mandatory
+
+O - optional
+
+
+Major Profile Version (X.Y)
+==================
```

Comment:
I missed this one ... title underline must be at least as long as the title, so add some more ``====``

Reply:
Thanks. Fixed it.